### PR TITLE
feat: request the renderer permissions to use web3 when using wallet connect

### PR DESCRIPTION
**Pull request by kuruk-mm**
# What? 
Fix decentraland/unity-renderer#1690

Test instructions: https://github.com/decentraland/unity-renderer/pull/1737

### CODE REVIEW

**Review by leanmendoza — CHANGES_REQUESTED** [review]
`this.parcelIdentity.land` can be undefined in Portable Experience (like smart wearables) even though currently the calls of web3 will fail in the `await this.assertHasPermissions([PermissionItem.USE_WEB3_API])` statement, there is no mechanism to grant web3 permissions to portable experience. I'm writing an example in the code that you can replicate it.
The PR looks good, I'd change that to avoid a crash in the future.

**Comment by kuruk-mm** [comment]
Do not merge until the Unity PR is merged to master, and the rollout is live.

**Review by leanmendoza — DISMISSED** [review]

File: packages/shared/apis/EthereumController.ts
Comment (menduz) [inline packages/shared/apis/EthereumController.ts:73]: @leanmendoza do we have a standardized way to get the sceneId for both PX and regular scenes?

File: packages/shared/apis/EthereumController.ts
Comment (gonpombo8) [reply packages/shared/apis/EthereumController.ts:59]: what about `this.parcelIdentity.getSceneId()` so we avoid repeating this logic multiple times ?
Also we can add the logic for px there. @leanmendoza @kuruk-mm

**Review by leanmendoza — APPROVED** [review]

**Review by leanmendoza — APPROVED** [review]

**Review by leanmendoza — APPROVED** [review]

**Review by gonpombo8 — APPROVED** [review]

**Comment by gonpombo8** [comment]
😄

**Review by menduz — APPROVED** [review]

**Review by pbosio — APPROVED** [review]

**Pull request merged**
